dv_utils/client.py

```diff
@@ -16,18 +17,71 @@
 
 class Client:
     """
-    Http client to interact with the Data-village API.
+    Http client to interact with the Datavillage API.
     """
+    DATA_PROVIDER_COLLABORATOR_ROLE_VALUE="DataProvider"
+
+    DATA_PROVIDER_COLLABORATOR_ROLE_VALUE="DataProvider"
+    DATA_CONSUMER_COLLABORATOR_ROLE_VALUE="DataConsumer"
 
     def __init__(self, settings: Settings = None):
         self.settings: Settings = settings or default_settings
+    
+    def get_data_descriptor(self, collaboration_space_id: str, data_descriptor_id: str, role: str):
+        """
+        Returns the data descriptor as json 
+        """
+        try:
+            list_participants = self.request(
+                f"/collaborationSpaces/{collaboration_space_id}/collaborators"
+            )
+            if role!=None:
+                filtered_participants = [x for x in list_participants if x["role"]==role]
+            else:
+                filtered_participants = [x for x in list_participants]
```

Comment:
`filtered_participants = [x for x in list_participants if role == None or role == x["role"]]`

Reply:
even better: `self.get_list_of_participants(collaboration_space_id, role)`

Reply:
✅ (I've removed the method, not used)